packages/components/input/src/input-element.tsx

```diff
@@ -21,9 +29,14 @@ interface InputElementOptions {
 export interface InputElementProps extends HTMLUIProps, InputElementOptions {}
 
 const InputElement = forwardRef<InputElementProps, "div">(
-  ({ className, isClick = false, placement = "left", ...rest }, ref) => {
+  (
+    { className, isClick = false, isClickable, placement = "left", ...rest },
```

Comment:
![badge](<https://img.shields.io/badge/review-q-blue.svg>)

Is the default argument for `isClickable` here not necessary?

Reply:
Hi, it is not necessary to set a default value here. If it is set, `isClick` will always be ignored.

Reply:
I see. I understand.